components/plurals/tests/fixtures/rules.json

```diff
@@ -121,10 +121,5 @@
     "rule": "e = 0 and i != 0 and i % 1000000 = 0 and v = 0 or e != 0..5 @integer 1000000, 1e6, 2e6, 3e6, 4e6, 5e6, 6e6, … @decimal 1.0000001e6, 1.1e6, 2.0000001e6, 2.1e6, 3.0000001e6, 3.1e6, …",
     "input": 1,
     "output": false
-  },
-  {
-    "rule": "c = 0 and i != 0 and i % 1000000 = 0 and v = 0 or c != 0..5 @integer 1000000, 1e6, 2e6, 3e6, 4e6, 5e6, 6e6, … @decimal 1.0000001e6, 1.1e6, 2.0000001e6, 2.1e6, 3.0000001e6, 3.1e6, …",
```

Comment:
Why did you delete this?

Reply:
because we don't support `e = 0` anymore. Or, if we do, there's no way to do a rountrip if I can't store C vs. E as two AST nodes, which is what you suggested me to remove.